Configure.cmake

```diff
@@ -310,6 +312,10 @@ if(CMAKE_C_COMPILER_ID MATCHES "(GNU|Clang)")
     string(CONCAT FLAGS_WALL ${FLAGS_WALL} " -Wno-psabi")
     set(FLAGS_ENABLE_NEON32 "-mfpu=neon")
   endif(CMAKE_C_COMPILER_ID MATCHES "GNU")
+
+  if (SLEEF_CLANG_ON_WINDOWS)
+    string(CONCAT FLAGS_WALL ${FLAGS_WALL} " -D_CRT_SECURE_NO_WARNINGS -Wno-deprecated-declarations")
```

Comment:
Why is this needed?

Reply:
Because clang references MSVC header files.
They have deprecation and security attributes for many functions, and many deprecation warnings are displayed without these options.

Reply:
Please explain this with a comment in the sources too.